config/config.go

```diff
@@ -136,21 +137,28 @@ type CheckPlugin struct {
 	NotificationInterval *int32
 	CheckInterval        *int32
 	MaxCheckAttempts     *int32
+	DisableAutoClose     bool
 }
 
 func (pconf *PluginConfig) buildCheckPlugin() (*CheckPlugin, error) {
 	err := pconf.prepareCommand()
 	if err != nil {
 		return nil, err
 	}
-	return &CheckPlugin{
+	plugin := CheckPlugin{
 		Command:              pconf.Command,
 		CommandArgs:          pconf.CommandArgs,
 		User:                 pconf.User,
 		NotificationInterval: pconf.NotificationInterval,
 		CheckInterval:        pconf.CheckInterval,
 		MaxCheckAttempts:     pconf.MaxCheckAttempts,
-	}, nil
+	}
+	if pconf.DisableAutoClose == nil {
+		plugin.DisableAutoClose = false
```

Comment:
Using `bool` (not `*bool`) in `pconf.DisableAutoClose` can simplify this assignment?